extensions/community_extensions/skadnetwork.md

```diff
@@ -848,28 +852,51 @@ Used for intermediary SSP to SSP/DSP to DSP connections where a full list SKAdNe
 }
 ```
 
+
 ## IABTL SKAdNetwork ID List Proposal
 
 “IABTL SKAdNetwork ID List” is a common list of networks, DSPs, Advertisers and others who support Apple’s SKAdNetwork API. This list would serve a similar purpose that the TCF 2.0 [Global Vendor List][8] serves to identify common IDs in a compact range. It can be used in addition to the distributed lists supplied by SSPs and Networks or in place of those.
 
+In the IABTL list model, the total list of SKAdNetworks on a device should be the union of the IABTL list and raw SKAdNetworks IDs supplied in the bid stream minus those in the exclude list
+
 ### Participant responsibilities
 
 The responsibilities of each participant when using the SKAdNetwork specifications are as follows (in addition to the responsibilities at the beginning of this document).
 
+#### IABTL responsibilities are to:
+1. Organize SKAdNetwork IDs as well as define an automated self-serve registration process
+    - Not validate any input data
+2. Perform releases in batches at “x” cadence to ensure as many partners publishers have the most up-to-date lists
```

Comment:
The lists will be one static list that will be updated with some frequency. It will not be versioned at release time. Likely it will be stored in github to see when the updates occurred but they should all remain on the same version so to speak until we change the schema, if that is even a thing.